scripts/publish/multiple_json_files.R

```diff
@@ -0,0 +1,41 @@
+publish.multiple_json_files <- function(viz) {
+  deps <- readDepends(viz)
+  args <- viz[["publish_args"]]
+  file_pattern <- args[["pattern"]]
+  
+  # unzip if it's a zip file
+  if(grepl(".zip", deps[["files_location"]])) {
+    # unzip and cache in a folder before publishing
+    extract_boundary_files(deps[["files_location"]], file_pattern, viz[["location"]])
+    paths_to_use <- list.files(viz[["location"]], full.names = TRUE)
+    
+  } else {
+    # paths are just the files in the passed in location if they aren't zipped
+    paths_to_use <- list.files(deps[["files_location"]], full.names = TRUE)
+  }
+  
+  for(fp in paths_to_use) {
+    
+    # create viz-like item to use in publish
+    viz_json <- list(location = fp,
```

Comment:
i don't think it affects functionality here, but would be nice to wrap this in a call to `as.viz` (there's a method specific to lists) so that the `viz` class gets added, in case `publish` ever starts doing something with that info.

```r
viz_json <- as.viz(list(location=fp, mimetype="application/json"))
```

Reply:
good call - I will include a commit for updating this as well!